flojoy/flojoy_python.py

```diff
@@ -84,6 +86,20 @@ def __init__(
         self.jobset_id = jobset_id
         self.node_type = node_type
 
+class cache_huggingface_to_flojoy(ContextDecorator):
+    """ Context manager to override the HF_HOME env var """
+    def __enter__(self):
+        self.old_env_var = os.environ.get("HF_HOME")
+        os.environ["HF_HOME"] = get_hf_hub_cache_path()
+        return self
+
+    def __exit__(self, *exc):
+        if(self.old_env_var is None):
+            del os.environ["HF_HOME"]
+        else:
+            os.environ["HF_HOME"] = self.old_env_var
+        return False
```

Comment:
Could you add a test that `HF_HOME` correctly gets set in the context of decorated functions (executed in other processes)?